crates/hc_sandbox/src/calls.rs

```diff
@@ -540,13 +547,22 @@ pub async fn install_app_bundle(cmd: &mut CmdRunner, args: InstallApp) -> anyhow
         agent_key,
         path,
         network_seed,
+        roles_settings,
     } = args;
 
+    let roles_settings = match roles_settings {
+        None => None,
+        Some(path) => {
+            let yaml_string = std::fs::read_to_string(path)?;
+            Some(serde_yaml::from_str::<RoleSettingsMap>(&yaml_string)?)
+        }
+    };
```

Comment:
Minor, but more idiomatic to start matching `Some` before `None`

```suggestion
    let roles_settings = match roles_settings {
        Some(path) => {
            let yaml_string = std::fs::read_to_string(path)?;
            Some(serde_yaml::from_str::<RoleSettingsMap>(&yaml_string)?)
        },
         None => None,
    };
```

Reply:
addressed :+1: